functional_tests/step_helpers/release_page_helpers.py

```diff
@@ -0,0 +1,287 @@
+
+
+def add_basic_table_block_under_pre_release_access(context: Context, data: bool = True, header: bool = True) -> None:
+    """Inserts a table block and fills with content."""
+    insert_block_under_pre_release_access(context, block_name="Basic table")
+    if header:
+        context.page.get_by_label("Table headers").select_option("column")
+    context.page.get_by_role("textbox", name="Table caption").fill("Caption")
+
+    if data:
+        context.page.locator("td").first.click()
+        context.page.keyboard.type("Name")
+        context.page.locator("td:nth-child(2)").first.click()
+        context.page.keyboard.type("Role")
+        context.page.locator("td").first.click()
+        context.page.keyboard.type("Heading")
+        context.page.locator("tr:nth-child(2) > td").first.click()
+        context.page.keyboard.type("Jack Smith")
+        context.page.locator("tr:nth-child(2) > td:nth-child(2)").first.click()
+        context.page.keyboard.type("Publisher")
+
+
+def add_description_block_under_pre_release_access(context: Context, index: int = 0) -> None:
+    """Inserts description and fills with text under pre-release access."""
+    insert_block_under_pre_release_access(context, block_name="Description", index=index)
+    context.page.get_by_role("region", name="Description *").get_by_role("textbox").fill("Description")
+
+
+def expect_text(
+    context: Context,
+    key: str | None,
+    texts: Sequence[str] | dict[str, str] | Mapping[str, str | Sequence[str]],
+) -> None:
+    if key:
+        texts = texts.get(key)
+    if isinstance(texts, str):
+        texts = [texts]
+    for text in texts:
+        expect(context.page.get_by_text(text)).to_be_visible()
+
+
+def add_date_change_log(context: Context) -> None:
+    change_to_release_date_section = context.page.locator("#panel-child-content-changes_to_release_date-section")
+    change_to_release_date_section.get_by_role("button", name="Insert a block").click()
+    change_to_release_date_section.get_by_label("Reason for change*").fill("Updated due to data availability")
+
+
+def add_release_date_text(context: Context) -> None:
+    fill_locator(context, RELEASE_DATE_TEXT, "March 2025 to August 2025")
+
+
+def add_next_release_date_text(context: Context) -> None:
+    fill_locator(context, NEXT_RELEASE_DATE_TEXT, "To be confirmed")
+
+
+def add_related_link(context: Context) -> None:
+    context.page.locator("#panel-child-content-related_links-content").get_by_role(
+        "button", name="Insert a block"
+    ).first.click()
+    choose_page_link(context.page, page_name="Home")
+
+
+def add_pre_release_access_info(context: Context) -> None:
+    add_basic_table_block_under_pre_release_access(context)
+    add_description_block_under_pre_release_access(context, index=1)
+
+
+def add_invalid_release_date_text(context: Context) -> None:
+    fill_locator(context, RELEASE_DATE_TEXT, "Invalid 5555")
+
+
+def add_invalid_next_release_date_text(context: Context) -> None:
```

Comment:
You can replace the value for the key with a lambda that performs the same action inside the FEATURE_ACTIONS dictionary